django-stubs/forms/utils.pyi

```diff
@@ -49,7 +48,7 @@ class ErrorDict(dict, RenderableErrorMixin):
     def as_data(self) -> Dict[str, List[ValidationError]]: ...
     def get_json_data(self, escape_html: bool = ...) -> Dict[str, Any]: ...
 
-class ErrorList(UserList, List, RenderableErrorMixin):
+class ErrorList(List[Union[ErrorList, Sequence[Union[str, Exception]]]], RenderableErrorMixin):
```

Comment:
It should be either `Sequence` or `List`.
List is just a mutable sequence (kind of).